new-lamassu-admin/src/pages/UserManagement/modals/ResetPasswordModal.js

```diff
@@ -0,0 +1,111 @@
+import { useMutation } from '@apollo/react-hooks'
+import { makeStyles } from '@material-ui/core/styles'
+import gql from 'graphql-tag'
+import React, { useEffect, useState } from 'react'
+
+import Modal from 'src/components/Modal'
+import { Info2, P, Mono } from 'src/components/typography'
+import CopyToClipboard from 'src/pages/Transactions/CopyToClipboard'
+
+import styles from '../UserManagement.styles'
+
+import Input2FAModal from './Input2FAModal'
+
+const CREATE_RESET_PASSWORD_TOKEN = gql`
+  mutation createResetPasswordToken($confirmationCode: String, $userID: ID!) {
+    createResetPasswordToken(
+      confirmationCode: $confirmationCode
+      userID: $userID
+    ) {
+      token
+      user_id
+      expire
+    }
+  }
+`
+
+const useStyles = makeStyles(styles)
+
+const ResetPasswordModal = ({
+  showModal,
+  toggleModal,
+  user,
+  requiresConfirmation
+}) => {
+  const classes = useStyles()
+  const [resetPasswordUrl, setResetPasswordUrl] = useState('')
+
+  const [createResetPasswordToken, { loading }] = useMutation(
+    CREATE_RESET_PASSWORD_TOKEN,
+    {
+      onCompleted: ({ createResetPasswordToken: token }) => {
+        setResetPasswordUrl(
+          `https://localhost:3001/resetpassword?t=${token.token}`
```

Comment:
We can't use localhost here. It has to be the same logic as the apollo url loader, might as well export that from some utils.